includes/admin/customers/class-customer-addresses-table.php

```diff
@@ -66,9 +66,10 @@ public function column_default( $item, $column_name ) {
 		switch ( $column_name ) {
 
 			case 'type' :
-				$value = ( 'primary' === $item['type'] )
-					? esc_html_e( 'Primary',   'easy-digital-downloads' )
-					: esc_html_e( 'Secondary', 'easy-digital-downloads' );
+				$value = ucwords( edd_get_address_type_label( $item['type'] ) );
+				if ( ! empty( $item['is_primary'] ) ) {
+					$value .= ' <span class="edd-chip">git ' . __( 'Primary', 'easy-digital-downloads' ) . '</span>';
```

Comment:
It also appears `$item['is_primary']` will never not be empty because it's not added to the normalized data here: https://github.com/easydigitaldownloads/easy-digital-downloads/blob/4bccc655e12c874ff20e923305c9c2f9a15977ab/includes/admin/customers/class-customer-addresses-table.php#L365-L378

Reply:
Confirmed that `Primary` isn't showing up in the UI for a primary address.